modules/core/src/test/java/org/apache/synapse/config/xml/MultiXMLConfigurationSerializerTest.java

```diff
@@ -0,0 +1,245 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.apache.synapse.config.xml;
+
+import org.apache.axiom.om.OMNode;
+import org.apache.synapse.config.SynapseConfiguration;
+import org.apache.synapse.core.axis2.ProxyService;
+import org.apache.synapse.mediators.TestUtils;
+import org.apache.synapse.mediators.base.SequenceMediator;
+import org.apache.synapse.mediators.template.TemplateMediator;
+import org.apache.synapse.registry.Registry;
+import org.apache.synapse.registry.SimpleInMemoryRegistry;
+import org.apache.synapse.startup.quartz.QuartzTaskManager;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This is the test class for MultiXMLConfigurationSerializer class.
+ */
+public class MultiXMLConfigurationSerializerTest {
+
+    private static final String SYNAPSE_XML = "synapse.xml";
+    private static final String KEY_DYNAMIC_SEQUENCE_1 = "dynamic_sequence_1";
+    private static final String KEY_DYNAMIC_ENDPOINT_1 = "dynamic_endpoint_1";
+    private static final String TEST_DIRECTORY_NAME = "serializeTestFolder";
+
+    private static final String DYNAMIC_ENDPOINT_1 =
+            "<endpoint xmlns=\"http://ws.apache.org/ns/synapse\">\n" +
```

Comment:
Line wrapping should happen before an operator

Reply:
fixed this